File: Front-end/src/app/Show.ts
import { ArenaLayout } from "./models/ArenaLayout";

export class Show {
    showId: number;
    price: number;
    seatRemaining: number;
    startTime: Date;
    date: Date;
    movieEventId: number;
    status:boolean;
    layout: ArenaLayout;
}

File: search-service/src/main/java/com/stackroute/eplay/search/domain/Show.java
package com.stackroute.eplay.search.domain;

import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.LocalTime;

import com.fasterxml.jackson.annotation.JsonFormat;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

//Pojo for Show
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Show {
	private int showId;
	private int seatRemaining;
	private BigDecimal price;
	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "HH:mm:ss")
	private LocalTime startTime;
	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
	private LocalDate date;
	private int movieEventId;
	private boolean status;
	// private ArenaLayout layout;
}
